src/main/java/no/nav/veilarboppfolging/service/OppfolgingService.java

```diff
@@ -458,6 +461,7 @@ private AvslutningStatusData getAvslutningStatus(Fnr fnr) {
                 .harYtelser(ytelserOgAktiviteterService.harPagaendeYtelse(fnr))
                 .underKvp(kvpService.erUnderKvp(aktorId))
                 .inaktiveringsDato(inaktiveringsDato)
+                .erSykmeldtMedArbeidsgiver(erSykmeldtMedArbeidsgiver)
```

Comment:
Kunne man heller tatt med `erIserv` i stede for `erSykmeldtMedArbeidsgiver`? Siden feilmelding i frontend er "Brukeren har aktiv status i Arena.", så holder det vel å vite at bruker er `!erIserv`?

Reply:
aha, jeg tenkte om det også...skal fikse nå...